test/e2e/ctx/ctx.go

```diff
@@ -137,6 +137,17 @@ func setDerivedFields(ctx *TestContext) error {
 		return fmt.Errorf("nil RESTClient")
 	}
 
+	if ctx.artifactsDir == "" {
```

Comment:
Could this poorly interact with some downstream only code: https://github.com/openshift/operator-framework-olm/blob/master/staging/operator-lifecycle-manager/test/e2e/e2e_test.go#L69?

Reply:
We should be in the clear there as we're building up the junit reporter for ginkgo vs. telling either of the provisioner implementation where to dump testing artifacts when configured.

Reply:
Also: we could bubble up that downstream-only change to make upstream backwards-compatible to eliminate any skew between implementations (which _might_ be relevant for any effort around ginkgo v1 -> v2)?